web/test/helpers/helpers-formation.tsx

```diff
@@ -88,6 +89,7 @@ export const preparePage = ({
   isAuthenticated,
   setRegistrationModalIsVisible,
   user,
+  searchOnly,
```

Comment:
I don't think we need to add this if we're not adding the tests for `searchOnly` back in as well, then this is never being used